docs/SIL.rst

```diff
@@ -5124,6 +5124,21 @@ different SSA value than the operand. This is important to ensure the
 correctness of ``ref_element_addr [immutable]`` for let-fields, because in the
 destructor of a class its let-fields are not immutable anymore.
 
+end_init_let_ref
```

Comment:
Same sort of question as with begin_deallocating. Would it make sense to add a verifier that would validate that ref_element_addr [immutable] even once Ownership SSA is lowered is after end_init_let_ref.

Reply:
Yes, it would make sense. I'll think about doing it in a follow up change